src/components/podcast/episode/Episode.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+/**
+ * Component for displaying the details of a podcast episode.
+ * @param {*} props podcast episode object 
+ */
+const Episode = ({episode}) => {
+    return (
+        <div><a href={episode.link}><h3>{episode.title}</h3></a> <hr /><div dangerouslySetInnerHTML={{ __html: episode.descriptionText }} /></div>
```

Comment:
Many teams would force you to do this another way since `dangerouslySetInnerHTML` is a potential security risk. From what I can see in your sample data, there isn't any html in your description text anyway so you should be able to just do `<div>{dangerouslySetInnerHTML}</div>` ... that said you could alternately run this through something like `DOMPurify`